BE/toDoListAPI/src/main/resources/application.yml

```
@@ -20,7 +20,7 @@ spring:
 
   jpa:
     hibernate:
-      ddl-auto: validate
+      ddl-auto: create-drop
```

Cuidado com isto, nao vais querer que create-drop va para producao!!!
```suggestion
      ddl-auto: validate
```